installCatalog.sh

```diff
@@ -40,7 +40,7 @@ DB_NAME="${4}alarmservice"
 APIHOST="$5"
 WORKERS="$6"
 LIMIT_CRON_FIELDS="${LIMIT_CRON_FIELDS}"
-ACTION_RUNTIME_VERSION=${ACTION_RUNTIME_VERSION:="nodejs:12"}
+ACTION_RUNTIME_VERSION=${ACTION_RUNTIME_VERSION:="nodejs:14"}
```

Comment:
This will probably not work, as a "nodejs:14" runtime will not be available. It could be either "nodejs:12" or "nodejs:16".

Reply:
Just checked, the value is overwritten at install time. Therefore this default is fine 👍 .